extensions/import/src/wizard/pages/fileConfigPage.ts

```diff
@@ -270,7 +270,8 @@ export class FileConfigPage extends ImportPage {
 		let connectionUri = await azdata.connection.getUriForConnection(this.model.server.connectionId);
 		let queryProvider = azdata.dataprotocol.getProvider<azdata.QueryProvider>(this.model.server.providerName, azdata.DataProviderType.QueryProvider);
 
-		let query = `SELECT name FROM sys.schemas`;
+		const escapedQuotedDb = this.databaseDropdown.value ? (<azdata.CategoryValue>this.databaseDropdown.value).name.replace(/]/g, ']]') + '.' : '';
+		const query = 'SELECT name FROM ${escapedQuotedDb}sys.schemas';
```

Comment:
It looks like you're using single quotes ( ' ) here instead of backticks ( \` ). You need to use backticks in order for the string.

Also you dropped bracket-quoting the name at some point. I tested this locally and it should work :

```
		const escapedQuotedDb = this.databaseDropdown.value ? `[${(<azdata.CategoryValue>this.databaseDropdown.value).name.replace(/]/g, ']]')}].` : '';
		const query = `SELECT name FROM ${escapedQuotedDb}sys.schemas`;
```